adapters/rubicon/rubicon.go

```diff
@@ -746,6 +747,82 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 	return requestData, errs
 }
 
+func getTpIdsAndSegments(eids []openrtb_ext.ExtUserEid) ([]rubiconExtUserTpID, []string, []error) {
```

Comment:
Can you please add test cases for this new method?

Reply:
Sure, added `TestOpenRTBRequestWithSpecificExtUserEids(...)` test.